radar.csv

```diff
@@ -24,3 +24,6 @@ Edge,adopt,platforms,true,"Edge is Microsoft's evergreen browser. We support it
 IE 11,adopt,platforms,true,"IE 11 is the default browser on Windows 10 and makes up approximately 10% of the visits to Redgate.com. We support it by default"
 IE 10 and under,retire,platforms,true,"We do not support IE10 and under"
 Raven DB,retire,tools,true,"We previously used Raven DB in DLM Dashboard but our experience was not positive."
+NSubsitute,adopt,languages and frameworks,true,"NSubstitute is our default mocking library of choice. We feel it's got less setup cost than Moq and requires fewer .Object calls"
+Moq,endure,languages and frameworks,true,"Moq is our legacy framework for Moq. We see no benefit in having a common mocking framework across solutions"
+Fakes,explore,techniques,true,"We've experimented using Fakes in the Redgate Usage Client - fakes used in the test can reduce the amount of mocking is required and can make the tests simpler"
```

Comment:
This section would probably benefit from a definition of what exactly we mean by fakes and mocking (personal preference is from here: https://github.com/testdouble/contributing-tests/wiki/Test-Double - in particular the two definitions for "Fake")